tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -799,6 +890,49 @@ def test_whisper_word_timestamps_batched(self):
         output = pipe(sample, batch_size=2)
         self.assertDictEqual(output, EXPECTED_OUTPUT)
 
+    @slow
+    @require_torch
+    def test_whisper_large_word_timestamps_batched(self):
+        pipe = pipeline(
+            task="automatic-speech-recognition",
+            model="openai/whisper-large-v3",
+            return_timestamps="word",
+        )
+        data = load_dataset("hf-internal-testing/librispeech_asr_dummy", "clean", split="validation")
+        sample = data[0]["audio"]
+
+        # not the same output as test_simple_whisper_asr because of chunking
+        EXPECTED_OUTPUT = {
+            "text": " Mr. Quilter is the apostle of the middle classes, and we are glad to welcome his gospel.",
+            "chunks": [
+                {"text": " Mr.", "timestamp": (0.0, 0.74)},
+                {"text": " Quilter", "timestamp": (0.74, 1.04)},
+                {"text": " is", "timestamp": (1.04, 1.3)},
+                {"text": " the", "timestamp": (1.3, 1.44)},
+                {"text": " apostle", "timestamp": (1.44, 1.74)},
+                {"text": " of", "timestamp": (1.74, 2.18)},
+                {"text": " the", "timestamp": (2.18, 2.28)},
+                {"text": " middle", "timestamp": (2.28, 2.5)},
+                {"text": " classes,", "timestamp": (2.5, 3.0)},
+                {"text": " and", "timestamp": (3.0, 3.4)},
+                {"text": " we", "timestamp": (3.4, 3.5)},
+                {"text": " are", "timestamp": (3.5, 3.6)},
+                {"text": " glad", "timestamp": (3.6, 3.84)},
+                {"text": " to", "timestamp": (3.84, 4.1)},
+                {"text": " welcome", "timestamp": (4.1, 4.4)},
+                {"text": " his", "timestamp": (4.4, 4.7)},
+                {"text": " gospel.", "timestamp": (4.7, 5.34)},
+            ],
+        }
+
+        # batch size 1: copy the audio sample since pipeline consumes it
```

Comment:
👀  - as in, mutates it? It shouldn't do that....

Reply:
Not sure to know why it does that, I've simply copied the logic from another test ([this one](https://github.com/huggingface/transformers/blob/15395df9d11d172197a69233418f0c49d7f00980/tests/pipelines/test_pipelines_automatic_speech_recognition.py#L886)) here!

Reply:
OK, let's open an issue to address this, so the work isn't forgotten but can be done in an separate PR. We should definitely avoid side-effect like this 😬

Reply:
Ok, I'll open an issue!